ports/servoshell/prefs.rs

```diff
@@ -119,16 +119,15 @@ pub fn default_config_dir() -> Option<PathBuf> {
 /// Get a Servo [`Preferences`] to use when initializing Servo by first reading the user
 /// preferences file and then overriding these preferences with the ones from the `--prefs-file`
 /// command-line argument, if given.
-fn get_preferences(opts_matches: &Matches, config_dir: &Option<PathBuf>) -> Preferences {
+fn get_preferences(opts_matches: &Matches, pref_dir: &Option<PathBuf>) -> Preferences {
```

Comment:
The config dir could also contain other configuration files besides prefs.json, so I wouldn't rename this variable here.